deploy/examples/pool.yaml

```diff
@@ -30,6 +30,8 @@ spec:
   #crushRoot: my-root
   # The Ceph CRUSH device class associated with the CRUSH replicated rule
   # For reference: https://docs.ceph.com/docs/master/rados/operations/crush-map/#device-classes
+  # If device classes are specified, ensure this property is added to every pool in the cluster,
```

Comment:
i guess, probably better to say : ensure that crush rule for pool has specified device class as well. Or recreate crush rule and set to pool.

Wdyt?

Reply:
Yeah, if the pool was already created, the crush rule will not be updated with the new device class. We probably need a new section on how to update the device class in [this doc](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/). It will involve several steps to recreate the crush rule. This sounds like a perfect command to add to the krew plugin.